app/models/manageiq/providers/hawkular/middleware_manager/event_catcher_mixin.rb

```diff
@@ -0,0 +1,67 @@
+# We use this to provide easy access to both the runner and specs
+module ManageIQ::Providers::Hawkular::MiddlewareManager::EventCatcherMixin
+  # extend ActiveSupport::Concern
+
+  # TODO: perhaps we need to block certain events.  Another way to do this would be to add a tag on events we want to
+  # suck into MiQ, and just fetch with the tag as criteria
+  INCLUDE_EVENTS = {'Hawkular Deployment' => '.*'}.freeze # some Hash of included events
+  EXCLUDE_EVENTS = {}.freeze # TODO: some Hash of exclusions for the included events
+
+  def event_monitor_handle(start_time = nil)
+    require 'middleware/events/middleware_event_monitor'
+    @event_monitor_handle ||= MiddlewareEventMonitor.new(@ems, start_time)
+  end
+
+  def reset_event_monitor_handle
+    @event_monitor_handle = nil
+  end
+
+  def stop_event_monitor
+    @event_monitor_handle.stop unless @event_monitor_handle.nil?
+  rescue => err
+    _log.error("#{log_prefix} Event Monitor error [#{err.message}]")
+    _log.error("#{log_prefix} Error details: [#{err.details}]")
+    _log.log_backtrace(err)
+  ensure
+    reset_event_monitor_handle
+  end
+
+  def monitor_events
+    event_monitor_handle.start
```

Comment:
you dont have to call this, if it doesnt do anything :smile:
have look here: https://github.com/ManageIQ/manageiq/blob/40c1166594e5985638d970710390e34da85a4782/gems/manageiq-providers-amazon/app/models/manageiq/providers/amazon/cloud_manager/event_catcher/runner.rb#L8-L16

`event_monitor_running` should be called once the first event is